### ISSUE TITLE
A small UX related quirk

### ISSUE THREAD

**Issue by HarikalarKutusu** [common-voice/common-voice#3264]
**Is your feature request related to a problem? Please describe.**
Suppose you are listening to recordings.
In the dashboard you select your language and validated clips from top contributors.
Then you hit Contribute and spend some time listening.
Then you hit the back button to see how far you have accomplished.
The interface tabs got reset to defaults (language=All, Top contributors recorded clips).
You start to click tabs to see you position.

**Describe the solution you'd like**
It would be nice to remember the previous tabs in a functional cookie for example.

Not a major issue but it is troublesome if you are spending much time in there.

**Comment by zaccolley** [comment]
This has been fixed in the 1.66.1 release: https://github.com/common-voice/common-voice/releases/tag/release-v1.66.1

Let us know how you get on with it. ❤️

**Comment by zaccolley** [comment]
Yes, you're right this isn't fixed, I think I misunderstood the request. Will reopen this PR, thanks @HarikalarKutusu

**Comment by HarikalarKutusu** [comment]
Thank you @zaccolley ,

To be clear: The marked areas are not remembered. The suggestion was that the user goes back where he/she left this page:

![image](https://user-images.githubusercontent.com/8849617/153942485-f29a4410-9108-4463-b29c-afc0816ead11.png)

Not a big issue tough, if it is the cause of the two other issues, this can be rolled back :)
